clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -699,4 +699,7 @@ public static void closeAll(Closeable... closeables) throws IOException {
             throw exception;
     }
 
+    public static int toPositive(int number) {
```

Comment:
Could we consolidate WindowedStreamPartitioner.toPositive and DefaultPartitioner.toPositive to use this function then?